### ISSUE TITLE
Crash on start

### ISSUE THREAD

**Issue by astubbs** [andreoliwa/logseq-doctor#73]
```
> ~/.local/bin/lsd outline /Users/antony/Downloads/italy\ trip\ august.md
Traceback (most recent call last):
  File "/Users/antony/.local/bin/lsd", line 8, in <module>
    sys.exit(main())
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/logseq_doctor/cli.py", line 34, in outline
    print(flat_markdown_to_outline(file.read()))
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/logseq_doctor/__init__.py", line 98, in flat_markdown_to_outline
    return mistletoe.markdown(markdown_contents, LogseqRenderer)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/__init__.py", line 21, in markdown
    return renderer.render(Document(iterable))
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/base_renderer.py", line 94, in render
    return self.render_map[token.__class__.__name__](token)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/base_renderer.py", line 204, in render_document
    return self.render_inner(token)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/base_renderer.py", line 109, in render_inner
    return ''.join(map(self.render, token.children))
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/base_renderer.py", line 94, in render
    return self.render_map[token.__class__.__name__](token)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/base_renderer.py", line 201, in render_thematic_break
    return self.render_inner(token)
  File "/Users/antony/.local/pipx/venvs/logseq-doctor/lib/python3.10/site-packages/mistletoe/base_renderer.py", line 109, in render_inner
    return ''.join(map(self.render, token.children))
AttributeError: 'ThematicBreak' object has no attribute 'children'
```

**Comment by andreoliwa** [comment]
Probably a duplicate of https://github.com/andreoliwa/logseq-doctor/issues/55.
Can you check if it's the same case?

Or paste some example of Markdown that causes the crash.

**Comment by andreoliwa** [comment]
I just released a [new version v0.2.1](https://github.com/andreoliwa/logseq-doctor/releases/tag/v0.2.1).
I'm closing this bug, please reopen it if the problem persists.
